deployment/helm/deploy-values.template.yaml

```diff
@@ -46,13 +46,13 @@ tiler:
     webConcurrency: "1"
 
   storage:
-    account_name: {{ tf.storage_account_name }}
-    account_key: {{ tf.storage_account_key }}
-    collection_config_table_name: {{ tf.collection_config_table_name }}
-    container_config_table_name: {{ tf.container_config_table_name }}
+    account_name: "{{ tf.storage_account_name }}"
+    account_key: "{{ tf.storage_account_key }}"
+    collection_config_table_name: "{{ tf.collection_config_table_name }}"
+    container_config_table_name: "{{ tf.container_config_table_name }}"
 
   deploy:
-    replicaCount: {{ tf.tiler_replica_count }}
+    replicaCount: "{{ tf.tiler_replica_count }}"
```

Comment:
Did you test the deployment of this? I'm not sure but I believe there might be an issue with k8s expecting an int value, and this would convert into a string. But I might be misremembering.

Reply:
Actually, when going in to make the change I noticed that most int values are already quoted, both literal and through template tags (see `port`, `proxy-buffer-number`, `maxWorkers`, etc) so I think I'll keep these as-is. Helm seems to tolerate them well enough.